doc/source/cluster/kubernetes/user-guides/kubectl-plugin.md

```diff
@@ -40,7 +40,16 @@ After installing the plugin, you can use `kubectl ray --help` to see the availab
 
 ## Example
 
-This example assumes you have a Ray cluster running on Kubernetes. See {ref}`RayCluster Quickstart <kuberay-raycluster-quickstart>` if you don't have a Ray cluster running on Kubernetes.
+Most of these example assumes you have a Ray cluster running on Kubernetes. See {ref}`RayCluster Quickstart <kuberay-raycluster-quickstart>` if you don't have a Ray cluster running on Kubernetes.
+
+### Get all Ray clusters
+
+```shell
```

Comment:
Maybe simply use plain text <code>```</code> if the code block containing logs because those logs do not need to be colored. Ditto for all other code blocks.

![image](https://github.com/user-attachments/assets/88885875-9a4b-4f22-b86d-19f65b8cf7ce)

Reply:
I changed all of the blocks to text except the code ones. PTAL